developers/bindings/index.md

```diff
@@ -966,7 +966,7 @@ mDNS discovery is implemented in the framework as `MDNSDiscoveryService`.
 To facilitate the development, binding developers only need to implement a `MDNSDiscoveryParticipant`.
 Here the developer only needs to implement four simple methods:
 
-- `getServiceType` - Defines the [mDNS service type](https://www.dns-sd.org/ServiceTypes.html).
+- `getServiceType` - Defines the [mDNS service type](http://www.dns-sd.org/ServiceTypes.html).
```

Comment:
Sad to have to revert https urls in 2022, but it seems correct unfortunately. 🤷‍♂️